src/jsonld/jobPosting.tsx

```diff
@@ -82,7 +83,8 @@ const JobPostingJsonLd: FC<JobPostingJsonLdProps> = ({
     "hiringOrganization" : {
       "@type" : "Organization",
       "name" : "${hiringOrganization.name}",
-      "sameAs" : "${hiringOrganization.sameAs}"
+      "sameAs" : "${hiringOrganization.sameAs}",
```

Comment:
```suggestion
      "sameAs" : "${hiringOrganization.sameAs}"
```

We need to add `,` to the beginning of the optional field below.

This is due to invalid JSON if the next line does not render as it is optional.